src/CMakeLists.txt

```diff
@@ -293,6 +297,24 @@ target_link_libraries(openroad
   ${CMAKE_THREAD_LIBS_INIT}
 )
 
+add_library(libopenroad
```

Comment:
should this also be linked against the optional import below (`zlib`, `cudd`, etc)